src/global.rs

```diff
@@ -34,17 +35,15 @@ extern "C" {
 #[repr(C)]
 pub struct Global<T> {
   value: Option<NonNull<T>>,
-  isolate: Option<NonNull<Isolate>>,
+  isolate_handle: Option<IsolateHandle>,
```

Comment:
This implies mutex locks for accessing globals - which in the single threaded case is unnecessary overhead.

Reply:
No mutexes are touched at any time while accessing globals.

Reply:
I see - well almost - `Mutex::new(())` is called once, like when you create your first global variable. Will that show up in dtrace? I would guess not.